docs/reference/api/storage/LittleFileSystem.md

```diff
@@ -12,181 +11,112 @@ for embedded systems, specifically microcontrollers using flash storage.
+
+We accounted for wear when we built the LittleFileSyste, and the underlying structure of the file system reactively mutates as the underlying storage develops errors throughout its lifetime. This results in a form of dynamic wear-leveling that extends the lifetime of the physical storage proportionally to the size of storage. With the LittleFileSystem, you can extend the lifetime of storage by increasing the size of storage, which is cheaper than upgrading the erase cycles on storage.
 
 ### Bounded RAM and ROM
 
-The scale of resources available to microcontrollers is foreign to the types
-of systems filesystems are normally targetting. A common trend of embedded
-Linux filesystems is RAM usage that scales linearly with the size of storage,
-which makes rationalizing RAM usage in a system difficult.
+The scale of resources available to microcontrollers is foreign to the types of systems file systems normally target. A common trend of embedded Linux file systems is RAM usage that scales linearly with the size of storage, which makes rationalizing RAM usage in a system difficult.
 
-The littlefs is built to work with a very limited amount of memory. Recursion
-is avoided and dynamic memory is limited to configurable buffers. At no point
-during operation does littlefs store an entire storage block in RAM. The result
-is small RAM usage that is completely independent of the geometry of the
-underlying storage.
+The LittleFileSystem works with a limited amount of memory. The LittleFileSystem avoids recursion and limits dynamic memory to configurable buffers. At no point during operation does the LittleFileSystem store an entire storage block in RAM. The result
+is small RAM usage that is independent of the geometry of the underlying storage.
 
 ---
 
-The little in littlefs comes from the focus on both keeping resource usage
-low and keeping the scope self-contained. Aside from the three targeted issues
-above, there is a heavy restriction against bloat in this software module.
-Instead, additional features are pushed separate layers in the powerful
-[block device API](TODO LINK ME) that drives the Mbed OS storage stack.
-This gives Mbed OS a powerful tool for remaining flexible as technology
-used by IoT devices develops.
+The "little" in the LittleFileSystem comes from the focus on both keeping resource usage low and keeping the scope self-contained. Aside from the three targeted issues above, there is a heavy restriction against bloat in this software module. Instead, additional features are pushed separate layers in the powerful [BlockDevice API](TODO LINK ME) that drives the Mbed OS storage stack. This gives Mbed OS a tool for remaining flexible as technology used by IoT devices develops.
```

Comment:
This could either be "The "little" in LittleFS" or "The "little" in the little file system".

But it's definitely not the class in this sentence.